execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -401,21 +412,33 @@ impl TransactionGenerator {
         block_size: usize,
         num_blocks: usize,
         transactions_per_sender: usize,
+        non_conflicting_txns: bool,
     ) {
         for _ in 0..num_blocks {
             // TODO: handle when block_size isn't divisible by transactions_per_sender
+            let mut random_account_generator: Box<dyn RandomAccountGenerator> =
```

Comment:
@sitalkedia should your changes to `run_transfer` go away?

Reply:
Yes, no need for any change in `run_transfer` function anymore since we are supporting that in `run_workload` function.